todo_app/app.py

```diff
@@ -1,11 +1,22 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
+
+from todo_app.data.session_items import get_items, add_item
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/', methods=['GET'])
```

Comment:
Nice - GET is the default but often explicitly calling things out is better than relying on people's knowledge of defaults!